extensions/submissions/upload-and-test.md

```diff
@@ -0,0 +1,236 @@
+
+We're now looking at the installation instructions for the Development environment. All we need to do here is copy the `<script>` tag in the box, so click the copy button.
+
+![](/images/getting-started/launch_installation_instructions_copy.png)
+
+Installation is completed by placing this single `<script>` tag inside the the `<head>` section of your document or site template.  Now just go visit your test site to test the behavior of your published Launch library.
+
+# 9) Test
+
+While validating your extension on your test page or site, there are a few console commands that will be helpful:
+
+- `_satellite.setDebug(true);` will put Launch into debug mode and output useful logging statements to the console.
+- The `_satellite._container` object contains all kinds of useful information about the deployed Launch library including details about the Build, Data Elements, Rules, and Extensions included.
+
+Ultimately your objective here is to test the functionality of the deployed Launch library to ensure that the code you wrote inside your extension package behaves as expected when Launch has compiled it into a library.
+
+When you discover changes that need to be made to your extension package, the iteration process is similar to the development process.
+
+1. Make changes to the code in your project
+2. Validate the changes with the Sandbox tool
+3. Use the Packager tool to create a new .zip package
+4. Use the Uploader tool to upload your new .zip package - you can follow the same instructions as above for the initial upload, but you'll notice that this time, because there is already an extension package of that name in development mode, this one will overwrite the other one instead of creating a new one.  If you want to save yourself some time by not entering credentials over and over again, you can look at the [reactor-uploader documentation](https://www.npmjs.com/package/@adobe/reactor-uploader) and pass in the arguments on the command line.
+5. We can skip the installation step this time around
+6. Modify resources - if you've changed the configuration for any of your extension components, you'll want to update those resources in the Launch UI
+7. Add your latest changes to your Library and build again
+8. Test some more
+
+# 10) Document
+
+Your [Exchange listing](#create-a-listing-in-the-exchange-partner-portal-required) is a great place for marketing and support information for your extension, but our Launch [Help Docs](https://docs.adobelaunch.com) are used every day by our customers. We encourage you to submit a pull request to [add your extension documentation](https://github.com/Adobe-Marketing-Cloud/reactor-user-docs/blob/master/extension-reference/3rd-party-extensions.md) into the Launch user docs. We also gladly accept [tutorials](https://docs.adobelaunch.com/launch-tutorials/contributing) where you can better illustrate the value of your extension for our mutual customers. Open source docs for the win! 🚀
```

Comment:
I don't think this is the link your intending here,  the tag is nowhere on the page
maybe - http://localhost:8001/extensions/submissions/create-listing/

Reply:
Yup, good catch.  And I also found a better way to do linking to anchor links.  New commit incoming.